### ISSUE TITLE
Lab 2 Example

### ISSUE THREAD

**Issue by krittschof** [DS4PS/cpp-524-sum-2021#10]
Good afternoon. After reading and reviewing the directives for Lab 2, I have a question regarding the example. It reports that there are 53% of male founders will slightly more experience; however, the 53% is under the female column and Yes row. Therefore, wouldn't that be female founders? Please help. Thanks much :)

**Comment by lecy** [comment]
That's correct, it should say "Female founders have slightly more experience".

> Female founders reported slightly more past experience (53 percent versus 47 percent of male founders), but the contrast was not significant at the alpha=0.05 level (chi-squre p-value 0.2277).

![image](https://user-images.githubusercontent.com/1209099/119566149-ac963500-bd5f-11eb-9628-c2b62ca5c968.png)

**Comment by krittschof** [comment]
Thank you!

**Comment by lecy** [comment]
Instructions have been updated to reflect!
